src/PanelTraits/Create.php

```diff
@@ -22,12 +25,12 @@ public function create($data)
         $data = $this->decodeJsonCastedAttributes($data, 'create');
         $data = $this->compactFakeFields($data, 'create');
 
-        // ommit the n-n relationships when updating the eloquent item
+        // omit the n-n relationships when updating the eloquent item
         $nn_relationships = array_pluck($this->getRelationFieldsWithPivot('create'), 'name');
         $item = $this->model->create(array_except($data, $nn_relationships));
 
         // if there are any relationships available, also sync those
-        $this->syncPivot($item, $data);
+        $this->createRelations($item, $data);
```

Comment:
As for the other issue, yes, deprecating the `syncPivot` method makes a lot of sense. Without removing it, of course, just not exposing it in the public API. People can still override the `create` or `update` methods if they want to implement their own approach.